Providers/nxOMSContainers/containers/plugin/filter_container.rb

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+
+module Fluent
+	require 'logger'
+
+	class ContainerFilter < Filter
+		Fluent::Plugin.register_filter('filter_container', self)
+		
+		config_param :enable_log, :integer, :default => 0
+		config_param :log_path, :string, :default => '/var/opt/microsoft/omsagent/log/filter_container.log'
```

Comment:
Make sure this path is always set to a multi-homed value.

Reply:
This needs modification to the docker-provider code itself. The OMI provider class somehow needs to know what workspace id its getting invoked for it to go look at wsid specific locations. For now this has to do.

Reply:
Ok; I would recommend making a comment for all spots in the code that will need that update when the time comes.